test/common.h

```diff
@@ -22,7 +22,10 @@
 # include <err.h>
 #else
 # define err(rc,msg,...) do { perror(msg); exit(rc); } while (0)
-# define errx(rc,msg,...) do { puts(msg); exit(rc); } while (0)
+# define errx(rc, msg, ...) do { \
+    fprintf(stderr, msg"\n", ##__VA_ARGS__);\
```

Comment:
Looks fine, but we really should have a similar fix for err too.

Reply:
yeah! good idea.